scripts/globals/combat/status_effect_tables.lua

```diff
@@ -113,24 +189,33 @@ xi.combat.statusEffect.isTargetImmune = function(target, effectId, actionElement
 end
 
 xi.combat.statusEffect.isTargetResistant = function(actor, target, effectId)
-    local modifierId = xi.combat.statusEffect.getAssociatedResistanceModifier(effectId)
+    local modifierId = xi.combat.statusEffect.getAssociatedResistTraitModifier(effectId)
     if modifierId == 0 then
         return false
     end
 
-    local resistancePower = target:getMod(modifierId) + target:getMod(xi.mod.STATUSRES)
-    if resistancePower == 0 then
+    local resistancePower = target:getMod(modifierId) + target:getMod(xi.mod.STATUSRES) + 5
+    if resistancePower <= 5 then
         return false
     end
 
-    local roll      = math.random(1, 100)
-    resistancePower = resistancePower + 5
-
     if actor:isNM() then
```

Comment:
We might want to add a TODO here, this isn't always true (resist terror from Sagasinger is not modified by NM)

Reply:
I wonder if that is something global for resistance mods from gear or a case by case thing.